### PR TITLE
Add provided dependency

### PR DESCRIPTION
**Pull request by Mrart**
Add provided dependency

### CODE REVIEW

**Review by caicancai — APPROVED** [review]
LGTM

**Comment by caicancai** [comment]
@Mrart Is there an English document so that we can update it together? I’m not sure this change is useful.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ tar -zxvf apache-streampark-2.2.0-incubating-bin.tar.gz
 
 ![Streampark Modify Run Configuration](/doc/image/streampark_modify_run_configuration.jpg)
 
-勾选`Add VM options`，填写参数`-Dapp.home=$path`，`$path`是我们刚刚复制的路径
+勾选`Add VM options`和`Add dependencies with “provided" scope to classpath`，填写参数`-Dapp.home=$path`，`$path`是我们刚刚复制的路径
```
Comment (wolfboys) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md:53]: 勾选 `Add VM options` 和 `Add dependencies with "provided" scope to classpath`，填写参数 `-Dapp.home=$path`，`$path`是我们刚刚复制的路径

注意下格式，中英连接部分要有空格，"provided" 左边引号不对。

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**